tests/behat/features/capabilities/event/filter-community-events.feature

```diff
@@ -0,0 +1,50 @@
+@api @javascript @event @stability @stability-1 @filter-community-events
+Feature: Filter community events
+  Benefit: Ability to filter events on All events page
+  Role: As a Verified
+  Goal/desire: I want to filter events that have all day checkbox activated
+
+  Scenario: Successfully use filters for ongoing and upcoming events with today's end date
```

Comment:
The contents of the test and what you're testing looks good! But we can fix a few readability things. Specifically prefer a clear separation of: arrange, act, assert to help readability. Suggestions on how to get there are below.

I'm also wondering what happens when the user is in other timezones and we probably should add test coverage for that (most of our users around the world are not in UTC)

Reply:
It seems we have some test https://github.com/goalgorilla/open_social/blob/main/tests/behat/features/capabilities/event/all-day-timezone.feature , although I'm not sure it's exactly what we need. What is your opinion on this?

Reply:
The test you referenced tests quite specifically for the display of dates and as outlined in my accompanying review comment I think it's also not sufficient on its own.

> We have a test that tests for the specific bug that we experience in the past in tests/behat/features/capabilities/event/all-day-timezone.feature. However, that test doesn't test what happens if the timezone is set to some + value (e.g. GMT+8) where I think we may have a bug if we manually set the time to 23:59:59. Before merging this I think we should at least add test coverage for that case so we can avoid a potential bug report and have confidence that it works :)

Working with dates can be difficult, so if we're testing something (like the community events filters) that depends on specific date boundaries, then I think we should write that test using multiple timezones too, so we know it works with different platform/user setting combinations.

Reply:
@Kingdutch Hi!
Ok, I think that makes sense :)
Thanks!